ext/json/json.c

```diff
@@ -35,6 +36,46 @@ PHP_JSON_API zend_class_entry *php_json_exception_ce;
 
 PHP_JSON_API ZEND_DECLARE_MODULE_GLOBALS(json)
 
+json_error_info_type json_error_info_data = {
+	NULL,
+	0,
+	0,
+	0
+};
```

Comment:
This could really use designated initializer, or at the bare minimum comments to document what each field is.

Also is there a reason this is a global?